Mathlib/Probability/Kernel/Condexp.lean

```diff
@@ -95,7 +95,7 @@ alias condexpKernel_apply_eq_condDistrib := condExpKernel_apply_eq_condDistrib
 instance : IsMarkovKernel (condExpKernel μ m) := by
   rcases isEmpty_or_nonempty Ω with h | h
   · exact ⟨fun a ↦ (IsEmpty.false a).elim⟩
-  · simp [condExpKernel, h]; infer_instance
+  · simpa [condExpKernel, h] using by infer_instance
```

Comment:
```suggestion
  · simpa [condExpKernel, h] using inferInstance
```
as before, optional and untested!

Reply:
Doesn't work either (would've been nice otherwise!)